must-gather/collection-scripts/gather

```diff
@@ -42,6 +42,7 @@ commands+=("get pv")
 commands+=("get sc")
 commands+=("get infrastructures.config -oyaml")
 commands+=("get subscription -n openshift-storage")
+commands+=("get csv -n openshift-storage")
```

Comment:
What about Petr's suggestion to use `-o yaml` ?
```suggestion
commands+=("get csv -n openshift-storage -o yaml")
```

Reply:
https://github.com/openshift/ocs-operator/blob/f6dcb364571ed3d7497a25a3698df679160e18ad/must-gather/collection-scripts/gather_ceph_resources#L115

Reply:
it already exists.

Reply:
hmm, but then why are we doing this at all? (I don't understand the difference/relationship between these various scripts...

Reply:
this is for only oc commands.  the changes which are in yaml uses inspect cmd.

Reply:
I still don't understand: the inspect command is also an oc command... Why do we need to get info about the csv twice once with `oc adm inspect` and once with `oc get`? (I don't know much about how must-gather works, so this may be dumb and feel free to ignore me...) And in general, why not use `-o yaml` for the "normal" oc commands?

Reply:
ocp must-gather uses inspect underlying.  So I try to follow that and use inspect only. This oc command was request from support so they can you some data which is in readeable formatt like describe.  This continued and now we have so many commands. But the target is still to capture data using inspect. I'll add a refactor PR which align these changes. there is bug which is targeted for 4.6.

Reply:
@obnoxxx when I am looking at the cluster and want to know it's installed an ok I am checking`oc get csv -n openshift-storage ` and it's output a lot. It will definitely be worth to have this human readable format in must gather data instead of go in big csv yaml file.

Reply:
@petr-balogh oh ok.  I thought you requested `-o yaml` - so you withdraw that request?

Reply:
Cause I thought we don't have that as well ;) But @crombus told me where it's collected.